utils/hct/hctdb.py

```diff
@@ -1406,6 +1406,7 @@ def populate_dxil_operations(self):
         # $o in a parameter type means the overload type
         # $r in a parameter type means the resource type
         # $cb in a parameter type means cbuffer legacy load return type
+        # $otpl{component} in a return type means the overload template shape with the specified component type
```

Comment:
I have a hard time reading overloads with this name.  What about just `$o_{component}` for overload, but with fixed component type?  I think it would be easier to read.

If necessary, this pattern could be extended to the `$x0` and `$x1` pattern, with `$x0_{component}`.